app.py

```
@@ -9,10 +9,14 @@
 import yaml
 from pathlib import Path
 import os
+import webbrowser
+import logging
```

Flake8 (7.2.0)

[error] redefinition of unused 'logging' from line 5

(F811)